WARNING: No test specified, defaulting to first test defined (xyz_deepstate_test_colsum_index_test)
TRACE: Initialized test input buffer with data from `/home/akhila/fuzzer_packages/fuzzedpackages/xyz/inst/testfiles/colsum_index/colsum_index_output/23d61a41ba6238524b5d9f475daa4c7604c1f03a.crash`
TRACE: Running: xyz_deepstate_test_colsum_index_test from /home/akhila/fuzzer_packages/fuzzedpackages/xyz/inst/testfiles/colsum_index/colsum_index_DeepState_TestHarness.cpp(13)
input starts
EXTERNAL: qs v0.23.4.

X values: 1.05415e-68 1.86447e-211 4.49059e+17 2.72551e-214 6.53679e-70 2.42643e+241 1.65687e+24

Missing value
indexes values: 246583936 -1079050344 49753534 37686913 -822602508 1591053158 878787584 742630714 -164119883 -423997720 -1317296461 -1145752183 -1387692533 503935315 123721430 123721430 814455830 1173028855 1722133025 1524607495 673715015 847551133 854406179 -435877353 -1868157482 -1859334665 -763946964 1472345215 -347617763 -882853123 1324228660 -215331965 -780576414 -1524213385 -1636719371 869823518 523975658 -1678382871 -486007082 862542340 -491559288 -624951432 816016739 2078376602
input ends
==375459==  If you believe this happened as a result of a stack
==375459==  overflow in your program's main thread (unlikely but
==375459==  possible), you can try to increase the size of the
==375459==  main thread stack using the --main-stacksize= flag.
==375459==  The main thread stack size used in this run was 8388608.
ERROR: Crashed: xyz_deepstate_test_colsum_index_test
ERROR: Test case /home/akhila/fuzzer_packages/fuzzedpackages/xyz/inst/testfiles/colsum_index/colsum_index_output/23d61a41ba6238524b5d9f475daa4c7604c1f03a.crash crashed
